WorkspaceServer/PackageRegistry.cs

```diff
@@ -157,7 +157,7 @@ private Task<IPackage> GetPackageFromPackageBuilder(string packageName, Budget b
             });
         }
 
-        public static PackageRegistry CreateForTryMode(DirectoryInfo project, PackageSource addSource = null)
+        public static PackageRegistry CreateForTryMode(IDirectoryAccessor directoryAccessor, PackageSource addSource = null)
```

Comment:
Maybe `projectDirectory` would be a more informative parameter name? 

Naming the parameter after the type isn't adding information. We know it's an `IDirectoryAccessor`. The parameter name can help explain what it will be used for in this context.

Reply:
Done. Thanks